This fixes the build against Boost 1.66

In file included from src/websockets/client/ws_client_wspp.cpp:28:
In file included from libs/websocketpp/websocketpp/config/asio_client.hpp:32:
In file included from libs/websocketpp/websocketpp/transport/asio/endpoint.hpp:32:
libs/websocketpp/websocketpp/transport/asio/connection.hpp:425:24: error: no matching function for call to 'make_shared'
            m_strand = lib::make_shared<boost::asio::strand>(
                       ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/usr/include/c++/v1/memory:4642:1: note: candidate template ignored: invalid explicitly-specified argument for template parameter '_Tp'
make_shared(_Args&& ...__args)
^

From 6b2e0480018530b616f61d5cdc786c92ba148bb7 Mon Sep 17 00:00:00 2001
From: John Hruby <johnh@vitrium.com>
Date: Tue, 26 Dec 2017 16:52:13 +0100
Subject: [PATCH] fixed strand

---
 Release/libs/websocketpp/websocketpp/transport/asio/connection.hpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Release/libs/websocketpp/websocketpp/transport/asio/connection.hpp b/Release/libs/websocketpp/websocketpp/transport/asio/connection.hpp
index 0be40f6b..395632c3 100644
--- libs/websocketpp/websocketpp/transport/asio/connection.hpp
+++ libs/websocketpp/websocketpp/transport/asio/connection.hpp
@@ -422,7 +422,7 @@ class connection : public config::socket_type::socket_con_type {
         m_io_service = io_service;
 
         if (config::enable_multithreading) {
-            m_strand = lib::make_shared<boost::asio::strand>(
+            m_strand = lib::make_shared<boost::asio::io_service::strand>(
                 lib::ref(*io_service));
 
             m_async_read_handler = m_strand->wrap(lib::bind(
